zingolib/src/lightclient/propose.rs

```diff
@@ -260,7 +260,7 @@ impl LightClient {
             // make it configurable?
             0,
         )
-        .map_err(ProposeShieldError::ShieldProposal)?;
+        .map_err(ProposeShieldError::Component)?;
```

Comment:
not sure about this name

Reply:
Me neither, but now it's not colliding with a type name.